src/strings/detect_anagrams.jl

```diff
@@ -0,0 +1,9 @@
+"""
+    detect_anagrams(subject, candidates)
+A function that checks if a list of words
+is an anagram or not of a subject word.
+"""
+function detect_anagrams(subject, candidates)
```

Comment:
Please add documentation for this function (see, for example, the [`knapsack`](https://github.com/TheAlgorithms/Julia/blob/main/src/knapsack/knapsack.jl) problem).